wazo_bus/resources/phone_number/event.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from ..common.event import TenantEvent
+from ..common.types import UUIDStr
+from .types import PhoneNumberDict
+
+
+class PhoneNumberCreatedEvent(TenantEvent):
+    service = 'confd'
+    name = 'phone_number_created'
+    routing_key_fmt = 'config.phone_number.created'
+
+    def __init__(self, phone_number: PhoneNumberDict, tenant_uuid: UUIDStr):
+        super().__init__(phone_number, tenant_uuid)
```

Comment:
not sure if there are precedents where that question arose, but do we really want to flood the bus with an individual event for each created phone number during bulk creation?

We could formulate the event to support bulking, providing a list of `phone_numbers: list[PhoneNumberDict]` instead.

Reply:
same can be considered for deleted and edit events, if we want to support bulk operations for those.

Reply:
I guess we need to think about how the events could be consumed to answer this, but I think it's easier to go from a bulk event to individual reactions per phone number, than having event-aggregating logic consumer-side.